### PR TITLE
Update Rule.php

### PR DESCRIPTION
**Pull request by prog98rammer**
update setParameters method

### CODE REVIEW

File: src/Rule.php [hunk 1]
```diff
@@ -52,18 +52,16 @@ public function getParameters()
         return $this->params;
     }
 
-    public function setParameters(array $params)
+    public function setParameters($key, $value = null)
```
Comment (Vandervir) [inline src/Rule.php]: but why?

Comment (Vandervir) [reply src/Rule.php]: Create another method to `setParameter` or so.

Function should make only one thing - in this case:

- add one key-value to paramers
- add array of parameters

File: src/Rule.php [hunk 2]
```diff
@@ -52,18 +52,16 @@ public function getParameters()
...
-        $this->params[$key] = $value;
+        if(is_array($key) && $value == null ) {
+            $this->params = array_merge($this->params, $key);
+        } else {
+            $this->params[$key] = $value;
+        }
         return $this;
     }
-
+
```
Comment (Vandervir) [inline src/Rule.php]: also unnecessary white characters

File: src/Rule.php [hunk 3]
```diff
@@ -109,4 +107,4 @@ protected function requireParameters(array $params)
         }
     }
 
-}
\ No newline at end of file
+}
```
Comment (Vandervir) [inline src/Rule.php]: CRLF line endings?

**Comment by emsifa** [comment]
What's this PR for?

**Pull request closed without merging**